src/main/java/step3/Operator.java

```diff
@@ -0,0 +1,58 @@
+package step3;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public enum Operator {
+	NOT_RANK(0, null, 0),
+	THREE(3, false, 5000),
+	FOUR(4, false, 50000),
+	FIVE(5, false, 1500000),
+	FIVE_BONUS(5, true, 30000000),
+	SIX(6, false, 2000000000);
```

Comment:
```suggestion
	SIX(6, false, 2_000_000_000);
```
큰 숫자들은 구분선을 넣어주면 좀 더 읽기 쉬워질 것 같아요 😄